### ISSUE TITLE
Filter on Name leaves visibility and status blank

### ISSUE THREAD

**Issue by johndoyle** [jayelkaake/enhancedgrid#29]
Hi - if you filter using the name column then the Visibility and Status columns go blank (Enhanced Grid v1.3.4.3 / Magento CE 1.9.1 / Sweet Tooth v1.8.6.2 / Better Store Search v3.3.0.0)

Thanks for your work on the Enhance Grid - Just as much fanciness to help us massively.
John

**Comment by jayelkaake** [comment]
Hey John!

Glad you like the extension. 

I'm actually unable to recreate the issue... could it be a module conflict of some sort? Some other modules rewrite the columns and such.

Here's my test just now:
![](http://g.recordit.co/uJoZCuNJiZ.gif)

**Comment by jayelkaake** [comment]
Just following up: There's likely a module conflict happening somewhere since some other modules sometimes rewrite the grid components. Try using a module conflict checker extension to see if there are any module conflicts. If you find out what it is please let me know!

**Comment by manojchowrasiya** [comment]
Hello jayelkaake,

I am facing same issue in magento 1.7.0.2, i checked and found that issue arrises incorrect datatype when we apply filter it is giving me float(i.e instead of 1 it is giving 1.0).

**Comment by webbrowser2014** [comment]
I'm having the exact same problem. I have four columns that are going blank after filtering by name. Status and three other attributes that I have added myself. I used this extension on an older version of Magento and had no difficulties. I have tried using a module conflict checker extension but this didn't indicate anything. What else should i try?

![website admin](https://cloud.githubusercontent.com/assets/22997616/19619232/feff4604-9859-11e6-855c-67f0f4ce163f.png)
